test/sensitivities/functional/reducedim.jl

```diff
@@ -54,8 +54,8 @@
                          randn(rng, 10, 10, 10), randn(rng, 10, 10, 10))
 
         # Issue #123
-        x6_ = collect(1:10)
-        tens = (fill(10.0, (10,)), fill(10.0, (10, 1)))
+        x6_ = float.(1:10)
```

Comment:
```suggestion
        x6_ = collect(1.0:10.0)
```

Reply:
why?

Reply:
I would say `collect` is clearer that you want a vector instead of a range

Reply:
I am convinced